### DISCUSSION TITLE
Unable to autoload freerdp-shadow-cli

### DISCUSSION THREAD

**Discussion by avandrval** [FreeRDP/FreeRDP#7591] [General]
0

Xrdp server on kali linux. Also there is installed freerdp2-shadow-x11 to start the shadow session. The shadow server is started with the freerdp-shadow-cli command in the terminal of a running xrdp session. And only so.Interactively on the command line with output.
I need it to run in the background from autoload along with the session of the user connected to the server. Partially, I solved it with a script:

#!/bin/bash freerdp-shadow-cli -auth -port:3389 -may-interact > /dev/null 2>&1 &

interactively, this script is executed and everything works, but if you add it to autoload (.zshrc file), then the script will only run when you open the terminal with your hands.

And so, experts, the question is: How can I make the script autoload?

**Comment by akallabeth** [comment]
You can only autostart it when the user is logged in. So use the autostart mechanism of your environment (`~/.config/autostart`?)

Note: 
1. using `freerdp-shadow-cli -auth` deactivates all authentication, so everyone that can reach your system can connect! This is dangerous!
2. `freerdp-shadow-cli -port:3389` is wrong, use `freerdp-shadow-cli /port:3389`. Also note, if your `xrdp` is running `3389` is most likely already in use!

**Reply by avandrval** [reply]
Thanks, I ended up setting it up like this:

https://stackoverflow.com/questions/70799831/unable-to-autoload-freerdp-shadow-cli
